spec/features/user_cannot_access_admin_privileges_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'Access', type: :feature do
+  create_user
+    feature 'user' do
+      it 'cannot access admin screens' do
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.